internal/querynode/segment_loader.go

```diff
@@ -913,7 +913,9 @@ func (loader *segmentLoader) checkSegmentSize(collectionID UniqueID, segmentLoad
 				usedMemAfterLoad += neededMemSize
 				usedLocalSizeAfterLoad += neededDiskSize
 			} else {
-				usedMemAfterLoad += uint64(funcutil.GetFieldSizeFromFieldBinlog(fieldBinlog))
+				if fieldID != common.RowIDField && fieldID != common.TimeStampField {
+					usedMemAfterLoad += uint64(funcutil.GetFieldSizeFromFieldBinlog(fieldBinlog))
```

Comment:
Why exclude RowID filed and timestamp filed when calculating memory usage?

Reply:
This demo still keeps row ID and timestamp in memory entirely